<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieSettingsWidget</class>
 <widget class="QWidget" name="MovieSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>625</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,1,0">
   <item>
    <widget class="QLabel" name="label_44">
     <property name="text">
      <string>Below you see the filenames which are used for loading and saving your movies. You can edit them as you like, if you want to use multiple files separate them by comma.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_45">
     <property name="text">
      <string>You can use the placeholder &lt;baseFileName&gt; which is the filename without extension.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_5">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Nfo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Poster</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Backdrop</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Logo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Clear Art</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Disc Art</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="movieNfo">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="moviePoster">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="movieBackdrop">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="movieLogo">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="movieClearArt">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="movieCdArt">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Banner</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>Thumb</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="movieBanner">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="movieThumb">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Movie outline</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="usePlotForOutline">
       <property name="text">
        <string>Use plot when outline is not available</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Movie Set Artwork</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboMovieSetArtwork">
       <item>
        <property name="text">
         <string>Artwork next to movies</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Separate artwork directory</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Movie Set Poster Filename</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="movieSetPosterFileName">
       <property name="text">
        <string>folder.jpg</string>
       </property>
       <property name="placeholderText">
        <string>folder.jpg</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Movie Set Fanart Filname</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="movieSetFanartFileName">
       <property name="text">
        <string>fanart.jpg</string>
       </property>
       <property name="placeholderText">
        <string>fanart.jpg</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelMovieSetArtworkDir">
       <property name="text">
        <string>Artwork directory</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="movieSetArtworkDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnMovieSetArtworkDir">
         <property name="text">
          <string>Choose directory</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
